corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -705,6 +708,11 @@ cloudCare.AppView = Backbone.View.extend({
                             lineNumbers: true,
                         });
 
+                        $instanceTab.off();
```

Comment:
I'm thankful for Ben
On Thu, Nov 26, 2015 at 9:47 PM Ben <notifications@github.com> wrote:

> In corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js
> <https://github.com/dimagi/commcare-hq/pull/9300#discussion_r46014323>:
>
> > @@ -705,6 +708,11 @@ cloudCare.AppView = Backbone.View.extend({
> >                              lineNumbers: true,
> >                          });
> >
> > +                        $instanceTab.off();
>
> this is ugly, but in order to not put this here, we need to do a pretty
> invasive refactor the debugger so i punted (debugger should belong in
> fullform-ui.js)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9300/files#r46014323>.
>